src/Misc/Master.cpp

```diff
@@ -1222,6 +1222,20 @@ bool Master::AudioOut(float *outr, float *outl)
     if(!runOSC(outl, outr, false))
         return false;
 
+    //request wavetables (ad synth) from MW if too many were consumed
```

Comment:
Likely isn't needed for _every_ master call, but that's a detail to worry about later. Seems like things are developing exactly according to your earlier plan.

Reply:
Good point, I'll leave that open for later.